app/models/identity.rb

```diff
@@ -4,14 +4,16 @@ class Identity < ApplicationRecord
 
   scope :enabled, -> { where(provider: Authentication::Providers.enabled) }
 
+  Authentication::Providers.available.each do |provider_name|
```

Comment:
mmm I thought about it but this would mean that a random script for maintenance or any other code could fail if an admin in the meantime has disabled one of the providers